adapters/taglib/taglib.go

```diff
@@ -122,16 +122,16 @@ func parseTIPL(tags map[string][]string) {
 	addRole := func(currentRole string, currentValue []string) {
 		if currentRole != "" && len(currentValue) > 0 {
 			role := tiplMapping[currentRole]
-			tags[role] = append(tags[currentRole], strings.Join(currentValue, " "))
+			tags[role] = append(tags[role], strings.Join(currentValue, " "))
 		}
 	}
 
 	var currentRole string
 	var currentValue []string
 	for _, part := range strings.Split(tipl[0], " ") {
-		if _, ok := tiplMapping[part]; ok {
+		if _, ok := tiplMapping[strings.ToLower(part)]; ok {
```

Comment:
Can't we just set the correct dj-mix case in the `tiplMapping` map, and avoid the ToLower here? This logic is too fragile and is based on exact matches, making it case insensitive could increase the chances of false positives.

Ideally we should parse TIPL directly, and not the string representation, but this will have to wait.

Reply:
Picard specifically does DJ-mix. I have no idea what other tools use.